pkg/spire/test/fakeworkloadapi/workload_api_windows.go

```diff
@@ -0,0 +1,65 @@
+//go:build windows
+// +build windows
```

Comment:
Can we add a short blurb on why a windows version is required? e.g. do we need to use the winio pipe, or can we simplify this with basic tcp on localhost?

Reply:
This is the remanent of the copy of the fakeworkloadapi. I will remove it.